src/requirements.cpp

```diff
@@ -333,6 +334,29 @@ void requirement_data::check_consistency()
     }
 }
 
+template<typename T>
+void requirement_data::finalize( std::vector< std::vector<T> > &vec )
```

Comment:
YAGNI, certainly at least now now. It's likely large chunks of the crafting code will shortly be ripped out to support `AND`/`OR`